components/docs-chef-io/data/automate/api-static/02-meta-description.yaml

```diff
@@ -41,3 +41,17 @@ info:
+    |-------------|-----------------------|--------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+    | 200         | OK                    | Correct Id, Auth token and JSON body | The API worked as expected                                                                                                                                                                                                                                  |
+    | 400         | Bad Request           | Wrong JSON body                      | The JSON body contains a key or value missing or in some cases when there is no json.  In some APIs like `IAM Project Rules`, the API request body validation is given the highest preference. And will return this error even before the Auth token validation. |
+    | 401         | Unauthorized          | Wrong Auth token                     | Wrong or no auth token provided                                                                                                                                                                                                                             |
+    | 403         | Forbidden             | Unauthorized access to Resources     | Unauthorised Resource ID is passed to the API.  The error is returned in case of no Auth token is passed for some APIs like `Node Export` and `ReportExport`.                                                                                                  |
+    | 404         | Not Found             | Wrong Id in URL                      | Wrong or no Id while creating, updating, retrieving, or deleting resources                                                                                                                                                                                   |
+    | 409         | Conflict              | Existing id                          | Existing resource id provided while creating resources                                                                                                                                                                                                      |
+    | 405         | Method Not Allowed    | Server does support the method       | Server knows the request but the target resource does not have a method to serve the request                                                                                                                                                                  |
```

Comment:
Remark: The server knows the request, but unfortunately, the target resource does not have a method to serve the request.